src/validations/rules/sap.sch

```diff
@@ -1263,6 +1292,7 @@
             <sch:assert
                 diagnostics="ipv6-has-non-placeholder-diagnostic"
                 feddoc:documentation-reference="OMB Mandate M-21-07"
+                fedramp:specific="true"
```

Comment:
IPv6 address specifications are not FedRAMP-specific.
And (separate from this PR)  the test is not properly constrained.